### ISSUE TITLE
ecs-conex check not appearing in github pull request checks

### ISSUE THREAD

**Issue by perrygeo** [mapbox/ecs-conex#129]
I've set up ecs-conex successfully: It shows up as an installed app in my repo's "Integration and Services" and docker images are getting built successfully on each commit.

The only issue is that the PR check is missing from the GitHub UI.

@arunasank noted that we should look at https://github.com/mapbox/ecs-conex/blob/8014ea9d2e791d54a930c8ad861173d10e911586/utils.sh#L42-L50

cc @rclark @arunasank - not urgent unless this is impacting other users. Thanks!

**Comment by arunasank** [comment]
@perrygeo yep, this is on my list, but I haven't found time to look at it yet. Thanks for flagging as something that's happening on newer PRs too. 🙇‍♀️ I'll post back here soon as I have updates.
